src/components/Form/FormProvider.js

```diff
@@ -272,6 +288,9 @@ const FormProvider = forwardRef(
 
                 return {
                     ...propsToParse,
+                    returnKeyType: shouldSubmitEdit ? 'go' : propsToParse.returnKeyType,
+                    blurOnSubmit: (isMultiline && shouldSubmitEdit) || propsToParse.blurOnSubmit,
```

Comment:
Anwering https://github.com/Expensify/App/pull/34471/files#r1461816468

I have to pass blurOnSubmit on multiline input only. So thats why I am calculating isMultiline again.

Reply:
Let's also hold on [this thing](https://github.com/Expensify/App/pull/34471/files#r1461807307), as the surrounding code might change if we go that route